### ISSUE TITLE
Plugin no longer working, only creates empty checkbox line

### ISSUE THREAD

**Issue by profucius** [lumoe/obsidian-rollover-daily-todos#81]
Hello, I am testing your plugin, and I do not think it works anymore. 

I have two files, both named in YYYY-MM-DD format. I am using Periodic notes and it is configured properly. I have some todo/checkbox lines in the old-date file which are unchecked. I trigger your command to rollover the todos, but all that happens is that a line is created into the new-date file as an unchecked checkbox with no text. The old-date file does not look changed at all. There are also no blank checkbox lines in the old-date file, so it could not be copying them as-is.

Obsidian v1.1.12.
No other plugins installed, besides Rollover Daily Todos and Periodic Notes.

If needed, I can provide a screen capture.

**Comment by tim-hilt** [comment]
@profucius thanks for letting us know! I think the majority of users use the daily notes core-plugin (which is working fine with the newest changes).

@lumoe that probably happens because of my changes. I didn’t test with periodic notes! Do you have that plugin configured and can take a look?

**Comment by tim-hilt** [comment]
@profucius if you know JavaScript you can also try submitting a PR! I can provide guidance if you need any

**Comment by tim-hilt** [comment]
@profucius one thing you could do to streamline debugging is to show screenshots of your settings for periodic-notes and rollover-daily-todos. This way we can try to reproduce your issue

**Comment by profucius** [comment]
I didn't realize it could be Periodic Notes, as you had stated in the Readme.md that your plugin works with "or the Periodic Notes plugin". And I do not know JS unfortunately, although I am generally code-adept enough to look at existing code and make recommended changes, if needed.

Sure, here are my current settings for Periodic Notes:
![image](https://user-images.githubusercontent.com/18254607/214141096-cbf4c588-6ed6-4cd9-a21e-5f93948a718a.png)

And here are my settings for Rollover Daily Todos:
![image](https://user-images.githubusercontent.com/18254607/214141165-639662bb-d0f3-44b0-b66c-e1a688da796d.png)

---

Update: Just as I was writing this comment, I found that suddenly the plugin is working again. I don't know what I did! I only closed and reopened Obsidian. Hmm, perhaps in doing so it flushed out something that was causing the issue. Anyway, I am keeping the screenshots above for you in case it helps in your development. But it is working now, sorry for the trouble!
